io.catenax.week_based_capacity_group/2.0.0/WeekBasedCapacityGroup.ttl

```diff
@@ -0,0 +1,318 @@
+   samm:dataType :Capacity .
+
+:Timestamp a samm:Characteristic ;
+   samm:preferredName "Timestamp"@en ;
+   samm:description "Point in time."@en ;
+   samm:dataType xsd:dateTimeStamp .
+
+:LinkedCapacityGroupsCharacteristic a samm-c:Set ;
+   samm:preferredName "Linked Capacity Groups Characteristic"@en ;
+   samm:description "Contains all capacity groups which are linked to the current one."@en ;
+   samm:dataType xsd:string ;
+   samm-c:elementCharacteristic :UUIDv4IdTrait .
+
+:BPNSTrait a samm-c:Trait ;
+   samm:preferredName "BPNS Trait"@en ;
+   samm:description "Trait to ensure data format for BPNS."@en ;
+   samm-c:baseCharacteristic ext-number:BpnsCharacteristic ;
+   samm-c:constraint ext-number:BpnsRegularExpression .
+
+:LinkedDemandSeries a samm:Entity ;
+   samm:preferredName "Linked Demand Series"@en ;
+   samm:description "Encapsulates information used to reference a specific demand series."@en ;
+   samm:properties ( :materialNumberCustomer [ samm:property :materialNumberSupplier; samm:optional true ] :customerLocation :demandCategory [ samm:property :loadFactor; samm:optional true ] ) .
+
+:Capacity a samm:Entity ;
+   samm:preferredName "Capacity"@en ;
+   samm:description "A supplier's realistically planned output per calendar week and material for a specific customer in a specific unit of measure, considering all positive or negative impacts on this capacity."@en ;
+   samm:properties ( :pointInTime :actualCapacity :maximumCapacity [ samm:property :deltaProductionResult; samm:optional true ] ) .
+
+:UUIDv4IdTrait a samm-c:Trait ;
+   samm:preferredName "UUIDv4 Trait"@en ;
+   samm:description "Trait to ensure data format for UUIDv4."@en ;
+   samm-c:baseCharacteristic ext-uuid:Uuidv4Characteristic ;
+   samm-c:constraint ext-uuid:Uuidv4RegularExpression .
+
+:materialNumberCustomer a samm:Property ;
+   samm:preferredName "Customer Material Number"@en ;
+   samm:description "Material identifier as assigned by customer. This material number identifies the material (as planned) in the customer's database."@en ;
+   samm:characteristic :MaterialNumber ;
+   samm:exampleValue "MNR-7307-AU340474.002" .
+
+:materialNumberSupplier a samm:Property ;
+   samm:preferredName "Supplier Material Number"@en ;
+   samm:description "Material identifier as assigned by supplier. This material number identifies the material (as planned) in the supplier's database. Used as informational field only, not for assigning demand series to capacity groups."@en ;
+   samm:characteristic :MaterialNumber ;
+   samm:exampleValue "MNR-8101-ID146955.001" .
+
+:customerLocation a samm:Property ;
+   samm:preferredName "Customer Location"@en ;
+   samm:description "The BPNS number of the site at which the customer needs the specified material for the referenced demand series."@en ;
+   samm:characteristic ext-number:BpnsTrait ;
+   samm:exampleValue "BPNS8888888888XX" .
+
+:demandCategory a samm:Property ;
```

Comment:
name of property and characteristic should not be same.

Reply:
renamed characteristic